src/components/JoinGroupForm/JoinGroupForm.tsx

```diff
@@ -127,13 +180,17 @@ export default function JoinGroupForm() {
         <>
```

Comment:
I kind of have this feeling that this ternary is a bit hard to read.
I am not sure if it has to do with indentation, but I got lost one I got here (as I didn't remember the conditional value since this wrapper is a bit long).
I just want to mention it so you give it a thought, but I think this is functional and that is our priority at the moment

Reply:
Ternary continuation in line 179

Reply:
The ternary is necessary as we are rendering two different views within the same modal. However, I agree that the wrapper is long. Maybe I can create a function that returns one of the two components based on the classInfo state